code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -411,6 +412,9 @@ var/global/list/damage_icon_parts = list()
 	var/skeleton = (SKELETON in src.mutations)
 	if(skeleton)
 		race_icon = 'icons/mob/human_races/r_skeleton.dmi'
```

Comment:
This line should be race_icon = icobase on the species datum, and the lower bit should be removed